ios/RNSScreenStack.m

```diff
@@ -553,23 +562,37 @@ - (BOOL)gestureRecognizerShouldBegin:(UIGestureRecognizer *)gestureRecognizer
   [self cancelTouchesInParent];
   return YES;
 #else
-  // custom animation should be always served by custom recognizers
+  if (topScreen.fullWidthGestureEnabled) {
+    // we want only `RNSPanGestureRecognizer` to be able to recognize when
+    // `fullWidthGestureEnabled` is set and only then
```

Comment:
rephrase

Reply:
`and only then` is not necessary since there is a comment about it already: https://github.com/software-mansion/react-native-screens/pull/1072/files#diff-1c3e13a1b0dcbaa5f7cff5f3ce12cc915fb509725faf693fb65cd23ceb49c4a9R595, so I will remove it.